content/developer/addons/theming-modules.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Theming Modules
+tags:
+- Theming Modules
+- Theming
+- Modules
+- Developers
+- Addons
+category: addons
+menu:
+  developer:
+    parent: addons
+    weight: 0
+
+---
+
+{{% cloudfeature %}}
+
+
+## Group Search Module
+
+The groups page has a search field, but if you with to move it somewhere, else, you can. There are 2 ways to add it to your theme.
+
+Note that the custom CSS class and the button contents is **optional** and will change the markup if used.
+
+### Adding Group Search Module - With Smarty
+
+```smarty
+{module name="GroupSearchModule" buttonContents="<span class='myCustomButtonContents'>Search Groups</span>" cssClass="customSearch"}
+``` 
+
+### Adding Group Search Module - With PHP
+
+```PHP
+if (Gdn::addonManager()->isEnabled('groups', Vanilla\Addon::TYPE_ADDON)) { // Make sure we have groups enablecd
```

Comment:
Typo at the end here "enablecd"

Also is this the way we want to do these checks now? In the past I've always seen it done with a class check. 

```php
if (class_exists("GroupSearchModule")) {
  // Instantiate module here
}
```

Reply:
According to @initvector this is the better way to check.